proto/process/agent.proto

```diff
@@ -176,8 +176,7 @@ message ECSContainerNetwork {
 message ECSContainerPort {
 	int32 containerPort = 1;
 	string protocol = 2;
-	string hostIp = 3;
```

Comment:
It is not necessary to set the hostIp at the container level, as all containers in the same task should have the same hostIp

Reply:
we can only know the private ip https://github.com/DataDog/agent-payload/blob/34accc1d6b77b4105419b110ddcb8dd5376019a2/proto/process/agent.proto#L170-L173

Reply:
I missunderstood it. It is the IP binding with the port. I've reverted the change.

- https://github.com/aws/amazon-ecs-agent/blob/master/agent/api/container/port_binding.go#L40
- https://github.com/aws/amazon-ecs-agent/blob/3d608ab0709f1d7be280ee60ce385abf127d857f/agent/handlers/v2/response.go#L221